applications/GeoMechanicsApplication/tests/test_transient_thermal.py

```diff
@@ -154,7 +154,149 @@ def test_unsteady_thermal_heat_flux_3D10N(self):
         temperature = test_helper.get_temperature(simulation)
         temp = temperature[124]
         self.assertAlmostEqual(1.2294110493528096, temp)
+ 
```

Comment:
this line has one white space. As I see other delimiting lines have only the end line symbol. Codacy does not complain on this line.

Reply:
removed.